site/en/blog/spa-view-transitions-land/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: 'layouts/blog-post.njk'
+title: SPA view transitions land in Chrome 111
+description: >
+  The View Transition API allows page transitions within single-page apps, and will later include multi-page apps.
+date: 2023-03-07
+authors:
+  - jakearchibald
+hero: 'image/CZmpGM8Eo1dFe0KNhEO9SGO8Ok23/nR4JNO8pelD4mGP3dQbr.jpg'
+alt: >
+  The Chrome DevTools animation timeline, showing a view transition.
+sharing_image: 'image/CZmpGM8Eo1dFe0KNhEO9SGO8Ok23/nR4JNO8pelD4mGP3dQbr.jpg'
+---
+
+The View Transition API lets you update the DOM in a single step, while generating an animated transition between the two states.
+
+<style>
+  .video-full-demo {
+    aspect-ratio: 1520 / 1054;
+  }
+</style>
+
+<figure>
+  {% Video
+    playsinline="true",
+    src="video/CZmpGM8Eo1dFe0KNhEO9SGO8Ok23/hgnJfPFUbGlucFegEEtl.mp4",
+    class="video-full-demo",
+    loop="true",
+    autoplay="true",
+    muted="true",
+    controls="true"
+  %}
+  <figcaption>Transitions created with the View Transition API. <a href="https://http203-playlist.netlify.app/">Try the demo site</a> – Requires Chrome 111+.</figcaption>
+</figure>
+
+This was a frequently-requested feature from developers, including me, and I think we've managed to land it in a way that balances good defaults with extensibility and customization. That sounds like we're patting ourselves on the back, but [developer](https://twitter.com/DavidKPiano/status/1428043899482824710) [feedback](https://twitter.com/jods16/status/1428434783441494023) was key to the design of this feature. [An earlier prototype](https://github.com/WICG/view-transitions/tree/92ea1f3e8cd5e16099b288838644d04c6cb08b61) of this feature was much less flexible, and folks (like you?) who took the time to play with the prototypes and provide feedback triggered a total rethink. Thank you!
+
+To get to grips with the feature, and play with some demos, [check out our guide](/docs/web-platform/view-transitions/). If there's something you feel isn't covered there, please [reach out to me on Twitter](https://twitter.com/jaffathecake), [Mastodon](https://mastodon.social/@jaffathecake), or [via email](mailto:view-transitions-api@chromium.org).
```

Comment:
You can add your mastodon link to your authors data now, if you like. It's in `_data/authorsData.json`